.github/workflows/build.yml

```diff
@@ -1,4 +1,4 @@
-name: Build
+name: build
```

Comment:
Can this be changed back to the original "Build"? Or was there some reason for this?

Reply:
a silly assumption 🦺